bin/rspecq

```diff
@@ -124,6 +124,20 @@ else
   redis_opts[:host] = opts[:redis_host]
 end
 
+# Use the RSpec parser to parse any command line args intended for rspec such
+# as `-- --format JUnit -o foo.xml` so that we can pass these args to rspec
+# while removing the files_or_dirs_to_run since we want to pull those from the
+# queue. OptionParser above mutates ARGV, so only options after `--`  or
+# non-flag arguments (such as files) will make it to this point.
+files_or_dirs_to_run = RSpec::Core::Parser.new(ARGV).parse[:files_or_directories_to_run]
+l = files_or_dirs_to_run.length
```

Comment:
Either use a better name for this shothand or simply use the entire `files_or_dirs_to_run.length`. It is not that bad. You don't save anything memory/speed-wise anyway.

Reply:
👍